src/components/BooksContainer.jsx

```diff
@@ -1,15 +1,33 @@
+import { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
-import { removeBook } from '../redux/books/bookSlice';
+import { removeBook, getBooks } from '../redux/books/bookSlice';
+import BookProgress from './Books/BookProgress';
+import Buttons from './Books/Buttons';
 
 const BooksContainer = () => {
+  const { isLoading } = useSelector((state) => state.books);
   const books = useSelector((state) => state.books.bookItems);
+  console.log(books);
```

Comment:
- [ ] please remove all the console.logs they make your code look unprofessional and they slow down performance.